client/src/components/SideBarHeader.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/core/styles";
+
+import MoreHorizIcon from "@material-ui/icons/MoreHoriz";
+import happyChatter from "../assets/happy-chatter.png";
+import { Typography } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  sideBarHeader: {
+    height: 100,
+    width: "85%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    "& div": {
+      display: "flex",
+    },
+  },
+  sideBarImgWrapper: {
+    "& span": {
+      position: "relative",
+      left: "-32%",
+      top: "75%",
+    },
+  },
+  sideBarImg: {
+    borderRadius: "100%",
+    overflow: "hidden",
+    width: "70px",
+    marginRight: "15px",
+  },
+  statusIcon: {
+    width: 12,
+    height: 12,
+    border: "solid white 1px",
+    borderRadius: "50%",
+  },
+  onlineIcon: { background: "#4DED84" },
+  offlineIcon: { background: "lightgray" },
+}));
+
+const SideBarHeader = (props) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.sideBarHeader}>
+      <div>
+        <div className={classes.sideBarImgWrapper}>
+          <img src={happyChatter} className={classes.sideBarImg} />
+          {/* background will depend on online status */}
+          <span
+            className={`${classes.statusIcon} ${classes.onlineIcon}`}
+          ></span>
+        </div>
+        <h4>Steven</h4>
```

Comment:
Also Typography here

Reply:
![Screen Shot 2020-11-13 at 9 45 38 AM](https://user-images.githubusercontent.com/18103497/99084529-1190f700-2595-11eb-8c9f-af049b5ef4cc.png)


I was using the h4 here because the element would be centered.  When I switch to Typography the element has all of this content on the bottom and I can't figure out what exactly is going on.  See screencap.

Reply:
Yeah, styling with Material UI can be tricky sometimes. I think what we want is alignContent on the container for the two? Try some of the suggestions from https://stackoverflow.com/questions/51940157/how-to-align-horizontal-icon-and-text-in-material-ui

Reply:
fixed as the div it was in was not set to alignItems: center